### PR TITLE
Split GraphQL query into fragments.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?
This pull request splits our single `PostGallery` query into fragments so we colocate data needs with the components that actually display them – in this case, `Post` and `ReactionButton`. These are new "forks" of `ReportbackItem` and `Reaction`, respectively.

### Any background context you want to provide?
This is not strictly necessary, but I think it helps show how this would be scalable as we continue rolling it out elsewhere in the application. One very cool little detail - we're able to auto-generate `PropTypes` from GraphQL fragments using the [graphql-anywhere](https://github.com/apollographql/apollo-client/tree/master/packages/graphql-anywhere) package's `propType` helper!

### What are the relevant tickets/cards?
[#155944555](https://www.pivotaltracker.com/story/show/155944555)

### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] Added screenshot to PR description of related front-end updates on **small** screens.
- [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
- [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlock.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
 
 import Card from '../Card';
 import Gallery from '../Gallery';
 import LoadMore from '../LoadMore';
-import { ReportbackItem } from '../ReportbackItem';
+import PostCard from '../PostCard/PostCard';
```
Comment (weerd) [inline resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlock.js]: 😏 "postcard"

Comment (DFurnes) [reply resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlock.js]: Hehe 😁 I'm very open to naming suggestions on both of these!

File: resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlockQuery.js [hunk 2]
```diff
@@ -4,26 +4,22 @@ import gql from 'graphql-tag';
 
 import CampaignGalleryBlock from './CampaignGalleryBlock';
 import PaginatedQuery from '../PaginatedQuery';
+import { postCardFragment } from '../PostCard/PostCard';
```
Comment (weerd) [inline resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlockQuery.js]: should these component live in the `/components/utilities`?

Comment (DFurnes) [reply resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlockQuery.js]: Done in 311340c!

File: resources/assets/components/utilities/PostCard/PostCard.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { propType } from 'graphql-anywhere';
+import gql from 'graphql-tag';
+
+import { Figure, BaseFigure } from '../../Figure';
+import ReactionButton from '../ReactionButton/ReactionButton';
+import { pluralize } from '../../../helpers';
+import './post.scss';
```
Comment (weerd) [inline resources/assets/components/utilities/PostCard/PostCard.js]: Small thing! Been trying to separate the SCSS imports from the JS ones. Mind adding a 👾 above it 😬

File: resources/assets/components/utilities/PostCard/PostCard.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { propType } from 'graphql-anywhere';
+import gql from 'graphql-tag';
```
Comment (weerd) [inline resources/assets/components/utilities/PostCard/PostCard.js]: Re-order from shortest to longest line!

File: resources/assets/components/utilities/ReactionButton/ReactionButton.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { propType } from 'graphql-anywhere';
+import classnames from 'classnames';
+import gql from 'graphql-tag';
```
Comment (weerd) [inline resources/assets/components/utilities/ReactionButton/ReactionButton.js]: Same here! Re-order from shortest to longest line!

File: resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlockQuery.js [hunk 6]
```diff
@@ -4,26 +4,22 @@ import gql from 'graphql-tag';
 
 import CampaignGalleryBlock from './CampaignGalleryBlock';
 import PaginatedQuery from '../PaginatedQuery';
+import { postCardFragment } from '../utilities/PostCard/PostCard';
+import { reactionButtonFragment } from '../utilities/ReactionButton/ReactionButton';
```
Comment (weerd) [inline resources/assets/components/CampaignGalleryBlock/CampaignGalleryBlockQuery.js]: And this one! Re-order from shortest to longest line!

**Pull request merged**